docs/book/src/reference/glossary.md

```diff
@@ -191,6 +216,10 @@ A generically understood combination of a kernel and system-level userspace inte
 # P
 ---
 
+### Patch 
```

Comment:
This is one that wasn't defined in a proposal, but seemed like it needed a definition here. @sbueringer WDYT?

Reply:
No objections